src/pages/CatIndex.js

```diff
@@ -1,13 +1,19 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
 
 class CatIndex extends Component {
-    render() {
-        return (
-            <div>
-                <h1>CatIndex Component</h1>
-            </div>
-        );
-    }
+  render() {
+    return(
+      <>
+        <h2>Dog Index</h2>
```

Comment:
Are you making dog tinder or cat tinder? Either is fine, but it would be a good idea to be consistent.

Reply:
lol you got us. We changed our minds mid-project with the full intent to correct the routing and coding syntax before we completed this project so that everything will be cohesive in the end. Thanks for noticing.